magma/lib/magma/attributes/collection.rb

```diff
@@ -7,7 +7,7 @@ def query_to_payload(link)
     end
 
     def query_to_tsv(value)
-      query_to_payload(value).join(", ")
+      value.join(", ")
```

Comment:
This addresses an unrelated issue that we discovered the other day?

Reply:
Yeah, this PR should really only be 1 commit and not include this code change, but I'm not sure how to get it looking clean. This change is already merged into [monoetna master](https://github.com/mountetna/monoetna/blob/master/magma/lib/magma/attributes/collection.rb). I think it's some issue with my workflow, since this change was in the standalone repository and had to get pulled in via `pull-subtrees`. I'll play around with it and see if I can get this to reflect just the one chnage.